content/blaze.md

```diff
@@ -0,0 +1,543 @@
+      return `no-${classname}`;
+    }
+  }
+});
+```
+
+You can also pass the output of a helper to a template inclusion or other helper. To do so, use brackets to show precedence:
+
+```html
+{{> todosItem (todoArgs todo)}}
+```
+
+Here the `todo` is passed as argument to the `todoArgs` helper, then the output is passed into the `todosItem` template.
+
+<h3 id="inclusion">Template inclusion</h3>
+You "include" a subtemplate with the `{{>` syntax. By default, the sub-template will gain the data context of the caller, although it's usually a good idea to be explicit. You can provide a single object as argument (as we did with the object returned by the `todoArgs` helper above), or provide a list of keyword arguments, as the `listShowPage` template does:
+
+```html
+{{> listsShow todosReady=Template.subscriptionsReady
+  list=(getFullList listIdOnly) todos=listIdOnly.todos}}
+```
+
+In this case, the `listShow` template can expect a data context of the form:
+
+```js
+{
+  todosReady: ...,
+  list: ...,
+  todos: ...
+}
+```
+
+<h3 id="helpers-in-tags">Helpers in tags</h3>
+We saw above that using a helper (or data context lookup) in the form `checked={{todo.checked}}` will add the checked property to the HTML tag if `todo.checked` evaluates to true. Also, you can directly include an object as part of an HTML to apply multiple properties at once:
+
+```html
+<a {{attributes}}>My Link</a>
+```
+
+// XXX: can we find an example of this in Todos?
+
+```js
+Template.foo.helpers({
+  attributes: () => {
+    return {
+      class: 'A class',
+      style: {background: 'blue'}
+    };
+  }
+});
+```
+
+<h3 id="rendering-html">Rendering pure HTML</h3>
+Although by default a mustance tag won't render any HTML, for XSS security reasons, you can render pure HTML with the triple-mustache: `{{{`.
+
+```html
+{{{myHtml}}}
+```
+
+```js
+Template.foo.helpers({
+  myHtml() {
+    return '<h1>This H1 will render</h1>';
+  }
+});
+```
+
+You should be extremely careful about doing this, and always ensure you aren't returning user-generated content (or escape it if you do!) from such a helper.
+
+<h3 id="block-helpers">Block Helpers</h3>
+A block helper, called with `{{#` is a helper that takes (and may render) a block of Spacebars. For instance, we saw the `{{#each in}}` helper above which repeats a given block of Spacebars once per item in a list. You can also render a *template* as a block helper, rendering it's content via the `Template.contentBlock` and `Template.eachBlock`. For instance, you could create your own `{{#if}}` helper with:
+
+```html
+<template name="myIf">
+  {{#if condition}}
+    {{> Template.contentBlock}}
+  {{else}}
+    {{> Template.elseBlock}}
+  {{/if}}
+</template>
+
+<template name="caller">
+  {{#myIf condition=true}}
+    <h1>I'll be rendered!</h1>
+  {{/myIf}}
+```
+
+<h3 id="builtin-block-helpers">Builtin Block Helpers</h3>
+There are a few builtin block helpers that are worth knowing about:
+
+<h4 id="if-unless">If / Unless</h4>
```

Comment:
Yes, that's true - you could manually add the ID in the URL. I just mean that the little link button doesn't show up. I'll need to modify my H4s to have good IDs then.